vue-frontend/src/components/portfolio/VideoSection.vue

```diff
@@ -184,7 +184,7 @@ section.video {
 }
 
 .background-video {
-  margin: 8% 30%;
+  margin: 6% 35%;
```

Comment:
Vertical margin should be in px

Reply:
but ti's nolonely video background circle so.